docs/DeveloperGuide.md

```diff
@@ -345,6 +375,33 @@ to the rest of the features that uses this method. This allows the UI to make us
 3 months fees as well. Although this results in increased coupling, with proper testing in place, we could mitigate the risk 
 as we ensure that changes in the `FeeUtil` method do not unintentionally changes the behaviour of the other feature.
 
+### General
```

Comment:
Actually not sure if we should put general above student or down here. Cos I tink we decided to put general above student in the UG, maybe putting in above might make it more standardise.

Reply:
Agreed, will make changes!